cli/pkg/kctrl/cmd/app/release/app_spec_builder.go

```diff
@@ -131,3 +138,16 @@ func (b *AppSpecBuilder) Build() (kcv1alpha1.AppSpec, error) {
 
 	return appSpec, nil
 }
+
+func (b *AppSpecBuilder) checkForErrorsAfterReconciliation(app kcv1alpha1.App, fakeClient *fakekc.Clientset) error {
+	existingApp, err := fakeClient.KappctrlV1alpha1().Apps(app.Namespace).Get(context.Background(), app.Name, metav1.GetOptions{})
+	if err != nil {
```

Comment:
Can fakeClient return err ?

Reply:
In this context I would not think so, but I would keep a check just in case something ever changes there.
I do not think we should leave errors unchecked